### PR TITLE
Fix for issue #632 - ensure find command is properly terminated

### PR DESCRIPTION
**Pull request by LRParser**
Now script executes correctly. Suggest that this script is also run/mentioned in the main README.md now - will create PR unless you disagree.

### CODE REVIEW

**Comment by lilleswing** [comment]
LGTM

**Comment by lilleswing** [comment]
@LRParser I would argue against mentioning the script in the README.md.  The purpose of the script is for people to play around with the datasets.  Most code paths internally which require data load the data lazily.

File: scripts/download_data.sh [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-# After running this script all data is in datasets
+# After running this script all data is in datasets. Assumes script is run from base directory
```
Comment (lilleswing) [inline scripts/download_data.sh]:
``` bash
Assumes script is run from deepchem/scripts
```
might be clearer?

Comment (rbharath) [inline scripts/download_data.sh]: +1 for @lilleswing's suggestion. @LRParser, could you make the change? Should be good to merge after.

**Comment by LRParser** [comment]
Thanks @lilleswing - added in comments per your feedback. Agree it doesn't need to be mentioned in README for now - was thinking more about adding it to a benchmarking sub-document so that lazy loading / IO doesn't influence benchmarking unduly - and this is the only way I've found to actually download Kaggle data. But as @rbharath suggests, we will remove Kaggle from benchmark for now, so it's a moot issue.

**Comment by rbharath** [comment]
@LRParser Thanks for the fix! And congrats on another merged in deepchem PR :+1: 

LGTM.

**Pull request merged**